client/flutter/lib/main.dart

```diff
@@ -1,11 +1,21 @@
+import 'dart:io';
+
+import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
-import './home_page.dart';
-import './constants.dart';
 import 'package:flutter_localizations/flutter_localizations.dart';
+
+import './constants.dart';
+import './home_page.dart';
 import 'generated/l10n.dart';
 
-void main() => runApp(MyApp());
+void main() {
+  /// Since macOS is not yet a supported platform in TargetPlatform we need
+  /// to override the default platform to one that is.
+  debugDefaultTargetPlatformOverride ??= TargetPlatform.iOS;
```

Comment:
This really should not go in our main code base.

Also https://github.com/flutter/flutter/pull/51519 has been merged, so we should be seeing it soon in flutter source.

 When that's the case devs can run the latest flutter to run the flutter app on macos, while our CI builds stay on stable (because they don't need to build the macos version anyway)